spec/spec_helper.rb

```diff
@@ -51,10 +52,18 @@ def self.run_all(reporter = nil)
   config.warnings = true
   config.raise_on_warning = true
 
+  # Execute a provided block with RSpec global objects (configuration,
+  # world, current example) reset. This is used to test specs with RSpec.
   config.around(:example) do |example|
-    real_world = RSpec.world
-    RSpec.instance_variable_set(:@world, RSpec::Core::World.new)
-    example.run
-    RSpec.instance_variable_set(:@world, real_world)
+    RSpec::Core::Sandbox.sandboxed do |sandbox_config|
+      # If there is an example-within-an-example, we want to make sure the inner
+      # example does not get a reference to the outer example (the real spec) if
+      # it calls something like `pending`.
+      sandbox_config.before(:context) { RSpec.current_example = nil }
+      RSpec::Rails.initialize_configuration(sandbox_config)
+      example.run
+    end
   end
+
+  config.include RSpec::Rails::FeatureCheck
```

Comment:
Was included in `Helpers` that didn't hesitate to include itself in `RSpec.configuration`.

Reply:
36 failures, 100% in generator specs, all `undefined method `type_metatag'`.
Besides `type_metatag`, generator specs use no methods from `FeatureCheck`.

I'll include this module exclusively to generator specs.

Frankly, I don't see a good reason for this method to exist:
```ruby
      def type_metatag(type)
        "type: :#{type}"
      end
```
but I'd prefer to avoid this unrelated refactoring here.